src/test/java/edu/kis/powp/jobs2d/events/SelectScaleCommandOptionListener.java

```diff
@@ -1,30 +1,31 @@
 package edu.kis.powp.jobs2d.events;
 
+import edu.kis.powp.jobs2d.command.CommandTransformationVisitor;
 import edu.kis.powp.jobs2d.command.ICompoundCommand;
-import edu.kis.powp.jobs2d.command.CommandTransformationScale;
 import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
 import edu.kis.powp.jobs2d.features.CommandsFeature;
+import edu.kis.powp.jobs2d.transformation.ScalePointTransformation;
 
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
 public class SelectScaleCommandOptionListener implements ActionListener {
```

Comment:
Now, SelectScaleCommandOptionListener and SelectCommandTransformationRotateOptionListener contain some code duplication and could be implemented as one listener.